GETTING_STARTED_GUIDE.md

```diff
@@ -25,380 +26,31 @@ Below are the set of steps to run OpenSearch and OpenSearch dashboards with the
-              },
-              "description": "A general tool to answer any question.",
-              "alias": "language_model_tool",
-              "include_output_in_agent_response": true,
-              "name": "QuestionSuggestor",
-              "type": "MLModelTool"
-            }
-          },
-          {
-            "id": "root_agent",
-            "type": "register_agent",
-            "previous_node_inputs": {
-              "agent_tool": "tools",
-              "register_model_2": "model_id",
-              "ml_model_tool": "tools"
-            },
-            "user_inputs": {
-              "parameters": {
-                "prompt": "Answer the question as best you can."
-              },
-              "app_type": "chatbot",
-              "name": "Root agent",
-              "description": "this is the root agent",
-              "tools_order": [
-                "agent_tool",
-                "ml_model_tool"
-              ],
-              "memory": {
-                "type": "conversation_index"
-              },
-              "type": "flow"
-            }
-          },
-          {
-            "id": "ppl_agent",
-            "type": "register_agent",
-            "previous_node_inputs": {
-              "TransferQuestionToPPLAndExecuteTool": "tools"
-            },
-            "user_inputs": {
-              "parameters": {},
-              "app_type": "query_assist",
-              "name": "PPL agent",
-              "description": "this is the PPL agent",
-              "type": "flow"
-            }
-          }
-        ],
-        "edges": [
-          {
-            "source": "register_model_2",
-            "dest": "rag_tool"
-          }
-        ]
-      }
-    }
-  }
-  ```
+6. Call the flow framework plugin to setup the cluster for the assistant.
+   1. See https://github.com/opensearch-project/flow-framework/tree/HEAD/sample-templates for sample templates.
```

Comment:
Should we suggest the `observability-chat-agent` template to follow for setting up the assistant? The user may not be sure which one to use for first time users.